backend/src/main/java/de/tobias/intestinalinspector/controller/PainController.java

```diff
@@ -49,6 +50,7 @@ public PainController(PainService painService, DateService dateService) {
         List<PainDto> painListToMap = map(listOfPain);
         Map<String, List<PainDto>> results = dateService.sortPainByDay(painListToMap);
         PainMapsDto painMapsDto = new PainMapsDto();
+
```

Comment:
This is very similar to what FoodController does. There is a chance you could make use of generics to extract the common behaviour into its own service. (-> refactoring ticket?)